packages/babel-plugin-transform-react-inline-elements/src/index.js

```diff
@@ -12,6 +12,13 @@ export default function ({ types: t }) {
     return t.isJSXAttribute(attr) && t.isJSXIdentifier(attr.name, { name: name });
   }
 
+  function getAttributeValue(attr) {
+    let value = attr.value;
```

Comment:
I tend to think that this is clearer how I have it because it indicates that you should pass an attribute node.